GitUI/CommandsDialogs/RevisionDiffControl.cs

```diff
@@ -1458,6 +1466,38 @@ private void ResetSelectedItemsWithConfirmation(bool resetToParent)
 
         private static bool RenamedIndexItem(FileStatusItem item) => item.Item.IsRenamed && item.Item.Staged == StagedStatus.Index;
 
+        private class ScriptOptionsProvider : IScriptOptionsProvider
+        {
+            private RevisionDiffControl _revisionDiffControl;
+
+            public ScriptOptionsProvider(RevisionDiffControl revisionDiffControl)
+            {
+                _revisionDiffControl = revisionDiffControl;
+            }
+
+            IReadOnlyList<string> IScriptOptionsProvider.Options => new[] { "SelectedFiles", "LineNumber" };
```

Comment:
This will unnecessarily allocate new arrays.
```suggestion
            IReadOnlyList<string> IScriptOptionsProvider.Options { get; } = new[] { "SelectedFiles", "LineNumber" };
```